### PR TITLE
Include "empty" privacy manifest file

### LINKED ISSUE #1529: xcprivacy file for this package
As per the new guidelines Third-party SDKs need to provide their own privacy manifest files. If this package doesn't have. Please let me know why it is not there and if the app review fails how can we justify this.

### PR DESCRIPTION
**Pull request by tbkka**
Include it as a resource in both SwiftProtobuf library and SwiftProtobufPluginLibrary when built via podspec or swiftpm

Resolves #1529

### CODE REVIEW

**Review by nabs-m — APPROVED** [review]
Looks good. I'm just not sure if you meant to reference `Sources/SwiftProtobufPluginLibrary/PrivacyInfo.xcprivacy` somewhere, or just `Sources/SwiftProtobuf/PrivacyInfo.xcprivacy` like you did.

**Review by thomasvl — APPROVED** [review]
Does the other Package-*.swift support resources?  i.e. - do we need to add it to the other file?

File: SwiftProtobuf.podspec [hunk 1]
```diff
@@ -16,6 +16,7 @@ Pod::Spec.new do |s|
   s.cocoapods_version = '>= 1.7.0'
 
   s.source_files = 'Sources/SwiftProtobuf/**/*.swift'
+  s.resource_bundle = {'SwiftProtobuf' => ['Sources/SwiftProtobuf/PrivacyInfo.xcprivacy']}
```
Comment (thomasvl) [inline SwiftProtobuf.podspec:19]: btw - Most things I've seen name this with `*_Privacy` (so `SwiftProtobuf_Privacy`), I'm not sure if it really matters or if you want to follow that convention.

File: Package.swift [hunk 2]
```diff
@@ -38,12 +38,14 @@ let package = Package(
     .target(
         name: "SwiftProtobuf",
         exclude: ["CMakeLists.txt"],
+	resources: [.copy("PrivacyInfo.xcprivacy")],
```
Comment (thomasvl) [inline Package.swift:41]: Oh, looks there might be a tabs vs spacing things going on in this file.

File: SwiftProtobuf.podspec (hunk 1 above)
Comment (tbkka) [reply SwiftProtobuf.podspec:19]: I really don't know.   The format I've used here seems to be commonly recommended in a few places.  I've yet to get any information from folks at Apple, though.

File: Package.swift (hunk 2 above)
Comment (tbkka) [reply Package.swift]: Ooops.  Fixed.

**Comment by tbkka** [comment]
> Does the other Package-*.swift support resources? i.e. - do we need to add it to the other file?

I don't think so.  Someone would have to be compiling an app for Apple's App Store with a Swift prior to 5.8 for it to make any difference.

**Comment by tbkka** [comment]
I presume we also need this in the 1.x branch?

**Pull request merged**